Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:stretch
+
+USER root
+RUN apt-get update -y && \
+	apt-get -y install build-essential libirrlicht-dev cmake libbz2-dev libpng-dev libjpeg-dev \
+		libsqlite3-dev libcurl4-gnutls-dev zlib1g-dev libgmp-dev libjsoncpp-dev git && \
+		apt-get clean && rm -rf /var/cache/apt/archives/* && \
+		rm -rf /var/lib/apt/lists/*
+
+COPY . /usr/src/minetest
+
+RUN	mkdir -p /usr/src/minetest/cmakebuild && cd /usr/src/minetest/cmakebuild && \
+    	cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_BUILD_TYPE=Release -DRUN_IN_PLACE=FALSE \
+		-DENABLE_GETTEXT=TRUE \
+		-DBUILD_SERVER=TRUE \
+		-DBUILD_CLIENT=FALSE \
+		-DENABLE_SYSTEM_JSONCPP=1 \
+		-DENABLE_SOUND=0 \
+		.. && \
+		make -j2 && \
+		rm -Rf ../games/minetest_game && git clone https://github.com/minetest/minetest_game ../games/minetest_game && \
+		make install
+
+FROM debian:stretch
+
+USER root
+RUN groupadd minetest && useradd -m -g minetest -d /var/lib/minetest minetest && \
+    apt-get update -y && \
+    apt-get -y install libcurl3-gnutls libjsoncpp1 liblua5.1-0 libluajit-5.1-2 libpq5 libsqlite3-0 \
+        libstdc++6 zlib1g libirrlicht1.8 libc6
```

Comment:
irrlicht is not actually needed at runtime for the server, so you may be able to omit it here

Reply:
i try to launch without it, because we linked against it and uses some irrlicht headers at compilation time.